javascript-modal/main.js

```diff
@@ -0,0 +1,14 @@
+var $openModal = document.querySelector('.open-modal-btn');
+var $noBtn = document.querySelector('.modal-btn');
+var $modalCont = document.querySelector('.modal-container');
+
+$openModal.addEventListener('click', openModal);
+$noBtn.addEventListener('click', closeModal);
+
+function openModal(event) {
+  $modalCont.style.display = 'block';
```

Comment:
Oh noooo, this is inline styling, exactly the same as using the `style` attribute on an element 💀 

Instead, I would make a class that you can add or remove to hide/show the element 🤓